lib/theme/colors.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+class AllAppColors {
+  static const MaterialColor labelLight = MaterialColor(0xFF36383A, {
+    100: Color(0xFFC5C5C7),
+    200: Color(0xFF8A8A8E),
+    300: Color(0xFF36383A),
+    400: Color(0xFF000000),
+  });
+  static const MaterialColor backgroundLight = MaterialColor(0xFFFFFFFF, {
+    100: Color(0xFFEEEEF3),
+    200: Color(0xFFF2F2F7),
+    300: Color(0xFFF6F6FB),
+    400: Color(0xFFFFFFFF),
+  });
```

Comment:
What's up with this 100 200 values? Shouldn't a hex code suffice?

Reply:
I looked up that these are swatches of the same color. My question is why do we need swatches? This is the first time I have seen a swatch at all in real world usage.

Reply:
ColorSwatch is basically a table of related colors, with a primary color. The shades are a slight modification of their primary color, with that shade index defining it lightness/darkness strength (as per the official docs of Flutter).
In apps, if there is some plan to add a new feature to the app, that uses a slightly lighter/darker color of the same color we have used at a place. Then instead of defining a new color hex individually, and using it, we can rather use a different shade of the existing color swatch (that we have customized as per our requirement, or use a pre defined color swatch).

Also, I believe using color swatches makes your app flexible to Color changes. Suppose you want to change the color theme of the app at some point. Here rather than changing the hex values of every color in your code, you can just change the primary color of the swatch you are using.

Here, the index values I have used are of <int> type. Instead of that, we can use String types as well (like lightest, lighter, light, dark, darker, darkest, etc). Int indeces were used in the Figma design, hence my choice here.

Reply:
Okay fair enough but the app's colors don't change that often and if it does, it IS very much a big change to do.
On the other hand, having code DOES add to complexity for the reader and the maintainer. I suggest you remove this and worry about color changes the day that happens.

Reply:
Ok. Used Color hexes instead of defining Color Swatches. Documented the Color definitions to the corresponding exact color shade as per the referenced figma design.